framework/jaif-inference-scripts/insert-annotations.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
```

Comment:
Would it be better for insert-annotations-to-source to be able to take a set of .jaif and .java files, so that this script is not necessary?